src/locales/fr/WCAG.json

```
@@ -0,0 +1,729 @@
+            "DETAILS": {}
+        },
+        "3.1.1": {
+            "TITLE": " Langue de la page",
+            "DESCRIPTION": "La langue par défaut de chaque page Web peut être déterminée par un programme informatique.",
+            "DETAILS": {}
+        },
+        "3.1.2": {
+            "TITLE": "Langue d’un passage",
+            "DESCRIPTION": "La langue de chaque passage ou expression du contenu peut être déterminée par un programme informatique sauf pour un nom propre, pour un terme technique, pour un mot dont la langue est indéterminée ou pour un mot ou une expression faisant partie du langage courant de la langue utilisée dans le contexte immédiat.",
+            "DETAILS": {}
+        },
+        "3.1.3": {
+            "TITLE": "Mots rares",
+            "DESCRIPTION": "Un mécanisme est disponible pour identifier la définition spécifique des mots ou expressions utilisés de manière inhabituelle ou de façon limitée, y compris les expressions idiomatiques et le jargon.",
+            "DETAILS": {}
+        },
+        "3.1.4": {
+            "TITLE": " Abréviations",
+            "DESCRIPTION": "Un mécanisme est disponible pour identifier la forme complète ou la signification d’une abréviation.",
+            "DETAILS": {}
+        },
+        "3.1.5": {
+            "TITLE": "Niveau de lecture",
+            "DESCRIPTION": "Lorsqu’un texte nécessite une capacité de lecture plus avancée que le premier cycle de l’enseignement secondaire après la suppression des noms propres et des titres, un contenu additionnel ou une version qui ne requiert pas de capacité de lecture supérieure au premier cycle de l’enseignement secondaire est disponible.",
+            "DETAILS": {}
+        },
+        "3.1.6": {
+            "TITLE": "Prononciation",
+            "DESCRIPTION": "Un mécanisme permet d’identifier la prononciation spécifique des mots dont la signification est ambiguë dans le contexte si leur prononciation n’est pas connue.",
+            "DETAILS": {}
+        },
+        "3.2.1": {
+            "TITLE": "Au focus",
+            "DESCRIPTION": "Quand un composant d’interface utilisateur reçoit le focus, il ne doit pas provoquer de changement de contexte.",
+            "DETAILS": {}
+        },
+        "3.2.2": {
+            "TITLE": "À la saisie",
+            "DESCRIPTION": "Le changement de paramètre d’un composant d’interface utilisateur ne doit pas provoquer de changement de contexte à moins que l’utilisateur n’ait été avisé de ce comportement avant d’utiliser le composant.",
+            "DETAILS": {}
+        },
+        "3.2.3": {
+            "TITLE": "Navigation cohérente",
+            "DESCRIPTION": "Dans un ensemble de pages, les mécanismes de navigation qui se répètent sur plusieurs pages Web se présentent dans le même ordre relatif chaque fois qu’ils sont répétés, à moins qu’un changement soit provoqué par l’utilisateur.",
```

```suggestion
            "DESCRIPTION": "Dans un ensemble de pages, les mécanismes de navigation qui se répètent sur plusieurs pages Web se présentent dans le même ordre relatif chaque fois qu’ils sont répétés, à moins qu’un changement soit initié par l’utilisateur.",
```

See https://www.w3.org/Translations/WCAG21-fr/#consistent-navigation
Please use the exact wording of the Authorized Translation

---

Idem. Voir plus haut.

---

See https://github.com/w3c/wai-wcag-em-report-tool/pull/127#discussion_r1402411577